maildump_runner/main.py

```diff
@@ -35,6 +35,10 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--smtp-ip', default='127.0.0.1', metavar='IP', help='SMTP ip (default: 127.0.0.1)')
     parser.add_argument('--smtp-port', default=1025, type=int, metavar='PORT', help='SMTP port (default: 1025)')
+    parser.add_argument('--smtp-auth', metavar='HTPASSWD', help='Apache-style htpasswd file for SMTP authorization. '
+                                                                'WARNING: do not rely only on this as a security '
+                                                                'mechanism, use also additional methods for securing '
+                                                                'MailDump instance, ie. IP restrictions.')
```

Comment:
+your (or this)

Reply:
Please explain.

Reply:
...for securing (your|this) MailDump instance...

Without that word the grammar isn't correct (it's a common mistake made by people with slavic/eastern-european mother tongues)